source/includes/fact-favorite-query.rst

```diff
@@ -0,0 +1,26 @@
+You can favorite a query so that you can find it later. If you
+load a favorited query, you can change it without
```

Comment:
[Note for throughout]

Can we use "favorite" instead of "favorited"? "Favorited" doesn't sound quite right to me.

```suggestion
load a favorite query, you can change it without
```